### PR TITLE
APEXCORE-815 Whitelist CVE-2016-6811

### PR DESCRIPTION
**Pull request by vrozov**

### CODE REVIEW

**Comment by vrozov** [comment]
test this please

**Comment by vrozov** [comment]
@tweise Please review. Likely I'll break it into two commits (upgrade maven and whitelist CVE).

File: pom.xml [hunk 1]
```diff
@@ -118,7 +114,7 @@
     <findbugs.failOnError>false</findbugs.failOnError>
     <dependency.check.failBuildOnCVSS>8</dependency.check.failBuildOnCVSS>
     <dependency.check.showSummary>true</dependency.check.showSummary>
-    <dependency.check.whitelist>${project.basedir}/../dependency-check-whitelist.xml</dependency.check.whitelist>
```
Comment (tweise) [inline pom.xml]: does this not work?

**Comment by tweise** [comment]
I would prefer we can commit the whitelist change separately in this PR, if possible at all.

**Review by tweise — APPROVED** [review]

**Pull request merged**